src/SixLabors.Fonts/FontReader.cs

```diff
@@ -197,4 +198,12 @@ public bool TryGetReaderAtTablePosition(string tableName, [NotNullWhen(returnVal
         reader = header?.CreateReader(this.stream);
         return reader != null;
     }
+
+    public void Dispose()
+    {
+        if (this.disposable)
```

Comment:
You need two `bool` fields.

1. `isDisposed` to prevent multiple calls.
2. `ownedStream` to specify ownership of the stream.